src/util/sss_ldap.c

```diff
@@ -301,6 +305,17 @@ static void sss_ldap_init_sys_connect_done(struct tevent_req *subreq)
                                          "Check for certificate issues.");
                 }
 
+                if (ticks_after_install > ticks_before_install) {
+                    ret = ERR_TLS_HANDSHAKE_INTERRUPTED;
+                    DEBUG(SSSDBG_CRIT_FAILURE,
+                        "ldap_install_tls failed: %s\n",
```

Comment:
This will look like:
```
ldap_install_tls failed: [Connect error] [unknown error]
ldap_install_tls failed: TLS handshake was interrupted
```
IMO, might be confusing.

I meant something like:
```
ldap_install_tls failed: [Connect error] [unknown error]
Assuming TLS handshake was interrupted
```
But I don't insist.